### PR TITLE
feat: split the accounts from the list into two

### LINKED ISSUE #8179: [Task]: split the accounts from the list into two
### Task description

Split the accounts from the list into two
1. the accounts outputs (ordered by `accountId`)
2. the implicit accounts outputs (ordered by `outputId`)

### PR DESCRIPTION
**Pull request by cpl121**
Closes #8179 

## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by marc2332 — CHANGES_REQUESTED** [review]
maybe try reducing the margin on top of "Pending accounts"
image

Also, it seems like I cannot scroll in the pending accounts section

image

File: packages/desktop/components/AccountManagementList.svelte [hunk 1]
```diff
@@ -27,42 +32,61 @@
...
-                {#each allOutputs as output, index}
-                    <ClickableTile onClick={() => onAccountClick(output)} selected={isSelected(output)}>
-                        <div class="flex flex-col space-y-4">
-                            <div class="flex space-x-2">
-                                <Text
-                                    type={TextType.h5}
-                                    color="gray-800"
-                                    darkColor="gray-500"
-                                    fontWeight={FontWeight.semibold}
+        <left-pane-container class="flex flex-col space-y-4 h-full scrollable-y">
```
Comment (evavirseda) [inline packages/desktop/components/AccountManagementList.svelte:35]: this should be fixed, it's cut off :police_car: 
image

suggestion--> `<list-wrapper class="flex flex-col space-y-2 px-2">`

**Review by evavirseda — CHANGES_REQUESTED** [review]
:pray:

**Review by evavirseda — APPROVED** [review]
nice job :rocket:

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**